### PR TITLE
fix(ios): missing support for density units on some properties

### PR DESCRIPTION
**Pull request by garymathews**
- Allow parsing of density units for various properties on iOS
  - `View.viewShadowRadius`
  - `padding` properties
  - `rect` properties
  - `shadow` properties
  - `Point` types

[JIRA Ticket](https://jira.appcelerator.org/browse/TIMOB-28145)

### CODE REVIEW

**Review by jquick-axway — APPROVED** [review]
CR: Pass

File: iphone/TitaniumKit/TitaniumKit/Sources/API/TiUtils.m [hunk 1]
```diff
@@ -394,10 +394,19 @@ + (UIEdgeInsets)contentInsets:(id)value
 {
   if ([value isKindOfClass:[NSDictionary class]]) {
     NSDictionary *dict = (NSDictionary *)value;
-    CGFloat t = [TiUtils floatValue:@"top" properties:dict def:0];
-    CGFloat l = [TiUtils floatValue:@"left" properties:dict def:0];
-    CGFloat b = [TiUtils floatValue:@"bottom" properties:dict def:0];
-    CGFloat r = [TiUtils floatValue:@"right" properties:dict def:0];
+
+    TiDimension tDimension = [self dimensionValue:@"top" properties:dict];
+    CGFloat t = t.value;
```
Comment (ewanharris) [inline iphone/TitaniumKit/TitaniumKit/Sources/API/TiUtils.m:399]:
```suggestion
    CGFloat t = tDimension.value;
```
Typo here I think

**Review by ewanharris — COMMENTED** [review]
iOS build looks to be failing because of the suggested change

**Comment by lokeshchdhry** [comment]
FR Passed

**Pull request merged**